lib/Settings/Admin.php

```diff
@@ -78,9 +80,28 @@ public function getForm(): TemplateResponse {
 			// Is it configured?
 			$key = array_search($backendId, $mountIds);
 			if (!empty($key) || $key === 0) {
```

Comment:
prefer to have more semantics boolean expression

 ```suggestion
            $isConfigured = !empty($key) || $key === 0;
			if ($isConfigured) {
```